src/dynamics/time_integration.jl

```diff
@@ -10,30 +10,57 @@ Base.@kwdef struct Leapfrog{NF} <: TimeStepper{NF}
 
     # OPTIONS
     "time step in minutes for T31, scale linearly to `trunc`"
-    Δt_at_T31::Float64 = 30
+    Δt_at_T31::Second = Minute(30)
 
     "radius of sphere [m], used for scaling"
     radius::NF = 6.371e6
 
+    "adjust Δt_at_T31 to the output_dt, so that it is rounded to closest"
+    "divisor of output_dt, ensures that output_dt is kept exactly in the output"
+    adjust_Δt_with_output::Bool = false 
+
+    "Output time step [hrs], needed if adjust_Δt_with_output is true"
+    output_dt::Second = Hour(6)
+
     # NUMERICS
     "Robert (1966) time filter coefficeint to suppress comput. mode"
     robert_filter::NF = 0.05
 
     "Williams time filter (Amezcua 2011) coefficient for 3rd order acc"
     williams_filter::NF = 0.53
 
-
     # DERIVED FROM OPTIONS    
     "time step Δt [s] at specified resolution"
-    Δt_sec::Int = round(Int,60*Δt_at_T31*(32/(trunc+1)))
+    Δt_sec::Second = get_Δt_sec(Second(Δt_at_T31), trunc, adjust_Δt_with_output, Second(output_dt))
 
     "time step Δt [s/m] at specified resolution, scaled by 1/radius"
-    Δt::NF = Δt_sec/radius
-    
-    "convert time step Δt from minutes to hours"
-    Δt_hrs::Float64 = Δt_sec/3600     
+    Δt::NF = value(Δt_sec)/radius  
```

Comment:
Especially given that the actual (scaled) time step inside the model isn't an integer anyway!

Reply:
both are NF now